### DISCUSSION TITLE
what are missing features in libre v izzy?

### DISCUSSION THREAD

**Discussion by idontveapc** [deckerst/aves#795] [Q&A] [answered]
i see the size differences and can't choose one.

**Comment by deckerst** [comment, marked as answer]
Check out this page for the differences:
https://github.com/deckerst/aves/wiki/App-Versions

I try to minimize differences between the versions, so it's mostly the same feature-wise.

As for the size, it's more or less the same once installed, but the apk size difference depends on whether the apk bundles everything so it can be installed on any device (whatever the architecture).

**Reply by idontveapc** [reply]
What's blunded shader?

**Reply by deckerst** [reply]
Bundled shaders may improve rendering performance the first time the app runs. These shaders are generally helpful, but not necessary. On some old or exotic environments they can prevent the app from launching, so that's why I also distribute some versions of the app without them.

For technical details: https://github.com/flutter/flutter/wiki/Reduce-shader-compilation-jank-using-SkSL-warm-up/bf8e395bac42e5249045e80d6cc92ad32b9839af

This shader warmup is only relevant with the legacy renderer Skia, and I will no longer bundle shaders when I switch to the new renderer Impeller.
